### ISSUE TITLE
LINE Login

### ISSUE THREAD

**Issue by tnylee** [FormidableLabs/react-native-app-auth#863]
I want to connect this library with [LINE Login v2.1](https://developers.line.biz/en/docs/line-login/integrate-line-login/) (it supports OpenID Connect). Is this currently possible?

Here's what I've tried but my app just crash:

```
const config = {
     issuer: 'https://access.line.me',
     clientId: 'xxxxxx',
     clientSecret: 'xxxxxx',
     redirectUrl: 'xxxxxx',
     scopes: ['openid', 'profile']
}

const authState = await authorize(config)
console.log('auth state ' + JSON.stringify(authState))
```

**Comment by januswel** [comment]
See https://github.com/FormidableLabs/react-native-app-auth/issues/800#issuecomment-1470488134
I've got the same situation, but above comment fixed the crash

If you still can't resolve it, you'd better show us the details of the crash

**Comment by robwalkerco** [comment]
I'm going to assume that your version of react native is 0.68+.
In that case this issue is resolved by #900
